i18n/ads-language-pack-de/CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 All notable changes to the "ads-language-pack-de" language pack will be documented in this file.
 
 ## [Released]
-* TBA - Release for Azure Data Studio 1.35
+* February 16, 2022 - Release for Azure Data Studio 1.35
```

Comment:
This isn't the correct release date

Reply:
I don't really think we need the date in this anyways, it's likely to just be incorrect some of the time and require annoying updates to fix and isn't something I think people would actually find useful.